config-dist.php

```diff
@@ -145,6 +145,7 @@
 Setting('FEATURE_FLAG_STOCK_PRICE_TRACKING', true);
 Setting('FEATURE_FLAG_STOCK_LOCATION_TRACKING', true);
 Setting('FEATURE_FLAG_STOCK_BEST_BEFORE_DATE_TRACKING', true);
+Setting('FEATURE_FLAG_STOCK_BEST_BEFORE_DATE_FIELD_AS_NUMBER', false);
```

Comment:
Maybe that better needs a short comment what it does...

Reply:
Updated the naming of this option and added a comment.